### ISSUE TITLE
Property declared in inherit

### ISSUE THREAD

**Issue by samreid** [phetsims/build-an-atom#98]
I noticed a property declared in the inherit call.  This almost certainly won't have the correct behavior because the Property instance will be shared across all ToElementProblemViews.  Link to the code issue forthcoming...

**Comment by samreid** [comment]
Problem site linked above.  @jbphet can you take a look?  Does this pattern occur elsewhere?

**Comment by samreid** [comment]
Same problem with the NumberAtom:

```js
  // Inherit from ProblemView.
  return inherit( ProblemView,
    CountsToElementProblemView,
    {
      periodicTableAtom: new NumberAtom(),
```

**Comment by jbphet** [comment]
This was resolved in 3d7e1761b096cbc19238a2ecbf76feba8db1235e, closing.
